### PR TITLE
KZOO-159: Handle outbound call transfer scenario

### PR DESCRIPTION
**Pull request by harenson**
5.0: https://github.com/2600hz/kazoo-callflow/pull/64

Before this change transfer or parking actions via /channels API were not working for outbound calls.

### CODE REVIEW

**Review by holy-batman — APPROVED** [review]

File: src/cf_route_win.erl [hunk 1]
```diff
@@ -300,13 +300,19 @@ maybe_start_account_recording(Call, FromNetwork, Endpoint) ->
...
 maybe_start_endpoint_recording(Call, <<"onnet">>=FromNetwork) ->
     maybe_start_endpoint_recording(Call, FromNetwork, cf_util:get_endpoint_id(Call));
 maybe_start_endpoint_recording(Call, <<"offnet">>=_FromNetwork) ->
     %% calls from offnet won't start endpoint recording unless an
     %% endpoint is dialed
     Call.
 
+-spec maybe_start_endpoint_recording(kapps_call:call(), kz_term:ne_binary(), kz_term:api_ne_binary()) -> kapps_call:call().
+maybe_start_endpoint_recording(Call, _FromNetwork, 'undefined'=EndpointId) ->
+    %% Transferred outbound call.
```
Comment (lazedo) [inline src/cf_route_win.erl:313]: `Transferred outbound call` how do you know that ?

**Review by lazedo — COMMENTED** [review]
looks like wrong way to address this

File: src/cf_route_win.erl (hunk 1 above)
Comment (harenson) [reply src/cf_route_win.erl:313]: Because it only sends `EndpointId = undefined` for that scenario.

Comment (jamesaimonetti) [reply src/cf_route_win.erl:313]: Not really required to bind EndpointId here, just log "no endpoint_id provided, not recording" or something

Comment (lazedo) [reply src/cf_route_win.erl:313]: if `EndpointId` is `undefined` then its an unauthorized call and shouldn't be handled

**Comment by harenson** [comment]
Thanks to Luis that found the issue was somewhere else and provided the fix, it can be found here https://github.com/2600hz/kazoo-ecallmgr/pull/76

Closing this PR.

**Pull request closed without merging**